libmscore/textbase.cpp

```diff
@@ -881,7 +881,7 @@ QFont TextFragment::font(const TextBase* t) const
       font.setItalic(format.italic());
       Q_ASSERT(m > 0.0);
 
-      font.setPointSizeF(m);
+      font.setPointSizeF(m * t->mag());
```

Comment:
It seems this has the effect of also scaling other text that has a "mag" associated with it.  This could be good, could be bad,  See https://musescore.org/en/node/312109

Reply:
Before this change <code>TextBase</code> simply ignored the scaling at all. The only way to the size of the text was changing the font size, which was the original "complaint" in [the original issue](https://musescore.org/en/node/308589). Now other text annotations are scaled with the note indeed. However, solving this is pretty simple (1 hour? I made already a simple prove of concept) but to decide which elements are supposed to scale with the notes and which element note, I'm not that sure that will be that easy.

Reply:
Sure, that's why I'm hoping for more discussion in the forum, to arrive at a consensus.  Then, I think, all we need to do is make sure we have appropriate "mag" functions defined for the element types.  For instance, for fingering, it could simply return the staff mag.

Reply:
That was indeed what I tried and that works fine 😃

Reply:
Did you ask @octophonie? He might have some thought on that.